release_automation.sh

```diff
@@ -34,12 +34,43 @@ MOBILE_REPO="wordpress-mobile"
 
 set -e
 
+SCRIPT_PATH="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+cd "$SCRIPT_PATH"
+source ./release_utils.sh
+
+
+# Check if script is up-to-date
+LOCAL_COMMIT=$(git rev-parse HEAD)
+git remote update
+DEVELOP_BRANCH=${1:-'develop@{upstream}'}
```

Comment:
I'm wondering in what cases we'd like to provide a different develop branch than `develop` 🤔 , do you foresee a particular situation where we would benefit from having this argument?

Reply:
I'm not quite sure what change you're suggesting here. My intention wasn't to allow us to provide a different `develop` branch, I basically just want to run `git rev-parse origin/develop` but I used the (super-readable 😉 ) `${1-'develop@{upstream}'}` instead of `origin/develop` to handle if someone had the remote named something other than `origin`. This is the first time I've used this `@{upstream}` syntax, so definitely let me know if you think there's a better way to go about this, I'm not committed to this approach at all.

Reply:
My comment was more related to the `${1` part of the command. I think what the following command `${1:-'develop@{upstream}'}` is doing is the following:
1. Take the first argument passed to the script as the `DEVELOP_BRANCH` value
2. If it's not provided then it will use the `develop@{upstream}` value.

That's why I asked about the potential situation where we might want to set a different value for `DEVELOP_BRANCH` by an argument (i.e. `./release_automation.sh different-develop-branch`).

Reply:
Ah, I understand now. Thanks for catching that, I wasn't intending to take the branch as an argument. I've pushed an update to remove that.

Reply:
Awesome, thanks for addressing it 🙇 .